### PR TITLE
SC-7198 - changes links in login page

### PR DESCRIPTION
**Pull request by virgilchiriac**
# Description

## Links to Tickets or other pull requests

- https://ticketsystem.schul-cloud.org/browse/SC-7198

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: locale/de.json [hunk 1]
```diff
@@ -380,11 +380,11 @@
 	"pages.impressum.title": "Impressum",
 	"pages.loginInstances.brb_link": "https://brandenburg.schul-cloud.org/login",
 	"pages.loginInstances.bundesland": "Wähle deine Cloud",
-	"pages.loginInstances.hpi_link": "https://schul-cloud.org/login",
+	"pages.loginInstances.hpi_link": "https://hpi-schul-cloud.de/login",
 	"pages.loginInstances.login": "Einloggen",
-	"pages.loginInstances.mintec_link": "https://schul-cloud.org/login",
+	"pages.loginInstances.mintec_link": "https://hpi-schul-cloud.de/login",
 	"pages.loginInstances.n21_link": "https://niedersachsen.cloud/login",
-	"pages.loginInstances.open_hpi_link": "https://open.schul-cloud.org/login",
```
Comment (janrenz) [inline locale/de.json:387]: this one stays the same

**Review by janrenz — CHANGES_REQUESTED** [review]

**Review by janrenz — DISMISSED** [review]

File: locale/de.json (hunk 1 above)
Comment (janrenz) [reply locale/de.json:387]: ok, in fact this one was changed , too

Comment (virgilchiriac) [reply locale/de.json:387]: brandenburg stays. This one changed

**Review by ABialoskorski — APPROVED** [review]

**Comment by ABialoskorski** [comment]
@schul-cloud-bot update screenshots

**Review by CeEv — APPROVED** [review]

**Review by pkleybolte — APPROVED** [review]

**Pull request merged**